project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.0.3
\ No newline at end of file
+sbt.version = 1.0.4
```

Comment:
Should we start using `1.1.0`?

Reply:
I'm not sure if the rest of the plugins we are using are already compatible with `1.1.x`.

Reply:
I created an issue some time ago to address it: https://github.com/47deg/sbt-org-policies/issues/910